src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -2792,7 +2790,7 @@ int32_t NaClSysImcRecvmsg(struct NaClAppThread         *natp,
   recv_hdr.iov_length = kern_nanimh.iov_length;
 
   recv_hdr.ndescv = new_desc;
-  recv_hdr.ndesc_length = NACL_ARRAY_SIZE(new_desc);
+  recv_hdr.ndesc_length = sizeof new_desc / sizeof *new_desc;
```

Comment:
Would prefer to always use parentheses with sizeof – makes things a bit easier to read and removes ambiguity.

Reply:
Sure, I can do that.